src/main/java/com/cs525/reversi/models/CellColor.java

```diff
@@ -0,0 +1,6 @@
+package com.cs525.reversi.models;
+
+public enum CellColor {
```

Comment:
@MustafaMosad `CellValue` will be a more accurate term... check the comment in `MatrixRow` class.... as of now I don't see any place where `CellColor` is used